source/llassetgen-rendering/CMakeLists.txt

```diff
@@ -0,0 +1,154 @@
+#
+# External dependencies
+#
+
+find_package(glm REQUIRED)
+find_package(glbinding REQUIRED)
+find_package(globjects REQUIRED)
+
+find_package(Qt5Core 5.2 REQUIRED)
+find_package(Qt5Gui 5.2 REQUIRED)
+find_package(Qt5Widgets 5.2 REQUIRED)
+find_package(Qt5OpenGL 5.2 REQUIRED)
+
+
+#
+# Executable name and options
+#
+
+# Target name
+set(target llassetgen-rendering)
+
+# Exit here if required dependencies are not met
+message(STATUS "Application ${target}")
+
+
+#
+# Sources
+#
+
+set(CMAKE_AUTOMOC TRUE)
+
+set(sources
+    main.cpp
+	
+	WindowQt.cpp
```

Comment:
There is a lot of mixing of tabs and spaces in this file. I think we use spaces in every other `CMakeLists.txt` and should probably spay consistent.

Reply:
Thanks. I didn't see that my Editor used tabs.